Sources/Controllers/TargetMenu/OAEditPointViewController.mm

```diff
@@ -309,34 +303,44 @@ - (void)setupIcons
     NSString *preselectedIconName = [self getPreselectedIconName];
 
     _poiIcons = [OAFavoritesHelper getCategirizedIconNames];
+    NSArray<NSString *> *categories = [OAFavoritesHelper getOrderedPoiIconsCategories];
+    
     if (_lastUsedIcons && _lastUsedIcons.count > 0)
```

Comment:
This code does not correspond to the Android version at all and is strange. Please revise